android/src/main/java/expo/modules/alternateappicons/ExpoAlternateAppIconsModule.kt

```diff
@@ -0,0 +1,51 @@
+package expo.modules.alternateappicons
+
+import android.content.ComponentName
+import android.content.pm.PackageManager
+import expo.modules.kotlin.Promise
+import expo.modules.kotlin.functions.Coroutine
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+class ExpoAlternateAppIconsModule : Module() {
+  override fun definition() = ModuleDefinition {
+    Name("ExpoAlternateAppIcons")
+
+    Constants(
+      "supportsAlternateIcons" to true
```

Comment:
Is there a way how to check the compatibility or is this feature generally supported on Android?

Reply:
As far as I know, `activity-alias` are available in Android from API level 1.